### PR TITLE
[FrameworkBundle][Secret] Add new force option to override existing secret in the vault

### LINKED ISSUE #38021: Why is the APP_SECRET stored in the .env file if secret / sensitive data shouldn't be stored there?
This has always felt weird to me, I understand not to store real sensitive data there, but as the 'fast track' book mentions not to store secrets in it, so, is "APP_SECRET" a good name for the variable?

### PR DESCRIPTION
**Pull request by 94noni**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | Related to https://github.com/symfony/recipes/pull/1005, Fix https://github.com/symfony/symfony/issues/38021#issuecomment-884556420
| License       | MIT
| Doc PR        | .

Here is a PR related to https://github.com/symfony/recipes/pull/1005#discussion_r714845582

In case we go for this option instead of always overrideing:
As it is a dedicated command, I do not know if I should handle the bc layer if command is called prio to sf v5.4.0

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Command/SecretsSetCommand.php [hunk 1]
```diff
@@ -49,13 +49,18 @@ protected function configure()
             ->setDescription(self::$defaultDescription)
             ->addArgument('name', InputArgument::REQUIRED, 'The name of the secret')
             ->addArgument('file', InputArgument::OPTIONAL, 'A file where to read the secret from or "-" for reading from STDIN')
+            ->addOption('force', 'f', InputOption::VALUE_NONE, 'Re-set an existing secret\'s value.')
```
Comment (OskarStark) [inline src/Symfony/Bundle/FrameworkBundle/Command/SecretsSetCommand.php:54]:
```suggestion
            ->addOption('force-override', 'f', InputOption::VALUE_NONE, 'Re-set an existing secret\'s value.')
```
Maybe it's a bit more clear?

Comment (94noni) [reply src/Symfony/Bundle/FrameworkBundle/Command/SecretsSetCommand.php:54]: @OskarStark I can reword of course 👍 
But I do prefer to wait for some answers on the PR meaning itself before continuing working on this :)

**Review by jderusse — COMMENTED** [review]
I'm not convinced by this feature...
`git status` and code review already prevent the issue

**Comment by 94noni** [comment]
@jderusse do you have another idea then to answer the referenced issue? thanks

**Review by nicolas-grekas — COMMENTED** [review]
If I understood correctly, this changes the default behavior so that an already set secret is not overridden unless --force is passed?

That doesn't make sense to me. I'm thus :-1: 

I don't know how the linked issue should be addressed.
Actually, I'm not even sure the linked issue is relevant at all.

**Comment by 94noni** [comment]
Totaly understood regarding your last comment on the issue
Thanks!

**Pull request closed without merging**